### ISSUE TITLE
No Timers display

### ISSUE THREAD

**Issue by dwalkiden** [AntonyCorbett/OnlyT#531]
**Describe the bug**
On some Android phones, we are having a problem with OnlyT showing the Timers list. We have verified the Wi-Fi connection to the K Hall PC but they don't show the Timers screen.

**To Reproduce**
3. See error

**Expected behavior**
Any help appreciated.

**Screenshots**

**Desktop (please complete the following information):**
 - Windows version 10 LATEST UPDATES ALL DONE.
 - Other important information about your environment

**Comment by AntonyCorbett** [comment]
Thanks for your report. Some questions, please:
1. Does the timer itself display properly on the Android device?
2. Is the “Timers” link visible? 
3. Have you tried clearing the cache on the Android browser?

**Comment by dwalkiden** [comment]
If turned landscape the timer click shows The Timers screen us just blank Havent tried clearing the browser cache as didnt know there was a connection with the App. Will try on SundayThanks for coming back to meDaveSent from my iPhoneOn 20 Dec 2024, at 18:44, Antony Corbett ***@***.***> wrote:﻿
Thanks for your report. Some questions, please:

Does the timer itself display properly on the Android device?
Is the “Timers” link visible?
Have you tried clearing the cache on the Android browser?

**Comment by dwalkiden** [comment]
Antony,

We tried that again today with a cleared cache but it still would not display the Timers screen detail, just a blank screen.

Not sure what could cause this, we have 2 others who were assigned to Timer but cannot get it to work.

 

Dave

**Comment by AntonyCorbett** [comment]
Just to confirm: are you using the “web clock” function rather than the remote timer app? Also, what version of OnlyT is installed on your PC?

**Comment by AntonyCorbett** [comment]
And I should have asked - is the “API” setting turned on

**Comment by dwalkiden** [comment]
AntonyWe are using the meeting timers of OnlyT. Not sure what you mean by web clock. we connect the app on the persons phone to the PC using the ip address and open port. The connection gets verified but the Timers screen for the meeting just shows blank. Other phones work no problem. The latest version of OnlyT is installed and updated each time it informs us of an update.DaveSent from my iPhoneOn 22 Dec 2024, at 13:11, Antony Corbett ***@***.***> wrote:﻿
Just to confirm: are you using the “web clock” function rather than the remote timer app? Also, what version of OnlyT is installed on your PC?

**Comment by dwalkiden** [comment]
Sorry i didnt see this other email.Not sure what the API is but other phones work and the connections are no issue except for some Android ones.DaveSent from my iPhoneOn 22 Dec 2024, at 16:00, Antony Corbett ***@***.***> wrote:﻿
And I should have asked - is the “API” setting turned on

**Comment by AntonyCorbett** [comment]
A couple of things:
1. The "web clock" is the ability to display the current timer on a web browser. See the Wiki for a full description. I assume this is what you are using. https://github.com/AntonyCorbett/OnlyT/wiki
2. In the OnlyT Settings page, there is a section called "Remote Apps". This must be enabled for the "Timers" page to work in the web clock. This is what I meant when I mentioned "API" in my previous comment - and it must have been confusing!

**Comment by dwalkiden** [comment]
AntonySorry if i have confused you. We are simply using OnlyT to allow someone in the audience to time each meeting part. The countdown is shown on a monitor by the sound desk.  No browsers are involved. Thats all we use. Its just that 3 people cannot get the display of the timers to show on their phone so they can be used to do the timing. Others have no issue with it, they see the Timers screen and are able to do it.Dave Sent from my iPhoneOn 22 Dec 2024, at 19:10, Antony Corbett ***@***.***> wrote:﻿
A couple of things:

The "web clock" is the ability to display the current timer on a web browser. See the Wiki for a full description. I assume this is what you are using. https://github.com/AntonyCorbett/OnlyT/wiki
In the OnlyT Settings page, there is a section called "Remote Apps". This must be enabled for the "Timers" page to work in the web clock. This is what I meant when I mentioned "API" in my previous comment - and it must have been confusing!

**Comment by AntonyCorbett** [comment]
Ah, you must be using one of the remote timer apps available for Android. Support should be available from the developer. The most popular one is here: https://play.google.com/store/apps/details?id=com.imattsolutions.onlyt

Please let me know how you get on.

**Comment by dwalkiden** [comment]
AntonySo onlyt remote is not yours?I thought we were using your one.They look identicalDaveSent from my iPhoneOn 23 Dec 2024, at 09:35, Antony Corbett ***@***.***> wrote:﻿
Ah, you must be using one of the remote timer apps available for Android. Support should be available from the developer. The most popular one is here: https://play.google.com/store/apps/details?id=com.imattsolutions.onlyt
Please let me know how you get on.

**Comment by dwalkiden** [comment]
what is the correct way/site link to download your OnlyT for Android phones please?

**Comment by AntonyCorbett** [comment]
Hello Dave. OnlyT is Windows desktop software only. However, it implements an API (programming interface) so that other developers can create software that interacts with it. This has allowed others to create mobile apps for controlling the timer. Please see the Wiki for more information. Here's the Facebook page for one developer who has created iOS and Android apps for OnlyT:

https://www.facebook.com/RemoteForOnlyT/

**Comment by dwalkiden** [comment]
Antony,

So I understand that you only provide the OnlyT program for Windows (KH PC) and that others have written software for Apple and Android…

 

Thus the problem is not that we have the 2 trying to connect together because they are compatible.

 

Thanks for the pointers. 

 

It’s strange that other phones both Apple and Android all connect but some cannot. 

We have to get help for this from the Android app writer

 

Hope I got that all correct. Thanks for your patience and help thus far.

 

Dave

**Comment by AntonyCorbett** [comment]
Yes, that’s it. Please let me know how you get on. Make sure you check the instructions given by the relevant remote control software.

**Comment by AntonyCorbett** [comment]
Closing - no response
